app/views/posts/_comment.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="single_comment">
+    <div class="comment_user_profile">
+    <img id="comment_user_photo" src= <%= comment.author.photo %> alt="user_image" />
```

Comment:
- [ ] @SekoViper kindly remove this photo section since its not in the wireframe design, as shown below the design requires the username and comment only
![image](https://user-images.githubusercontent.com/61408860/225419776-aa87c45a-b20e-48ac-a389-39eefc50d69a.png)